src/index.html

```diff
@@ -1,13 +1,310 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet"> 
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+      <div class="section__home">
+        <div class="section__home__left">
+          <div class="section__home__left__nav">
+            <div class="section__home__left__nav__logo">
+              <a href="#">Thrivetalk</a>
+            </div>
+            <div class="section__home__left__nav__menu">
+              <a href="#" class="section__home__left__nav__menu__item">Home</a>
+              <a href="#About" class="section__home__left__nav__menu__item">About</a>
+              <a href="#Services" class="section__home__left__nav__menu__item">Services</a>
+              <a href="#Blog" class="section__home__left__nav__menu__item">Blog</a>
+            </div>
+          </div>
```

Comment:
```suggestion
      <div class="section__home">
        <div class="section__home left">
          <div class="left__nav nav">
            <div class="nav__logo">
              <a class="logo" href="#">Thrivetalk</a>
            </div>
            <div class="nav__menu">
              <a href="#" class="nav__item">Home</a>
              <a href="#About" class="nav__item">About</a>
              <a href="#Services" class="nav__item">Services</a>
              <a href="#Blog" class="nav__item">Blog</a>
            </div>
          </div>
```
Шото классы не по БЭМу, должны быть вот так примерно
Ну и не понятно что значит left, вдруг этот блок переедет направо? Надо будет менять название класса. Задавай названия так, чтобы они описывали то что находится внутри, а не положение блока на странице